azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_mqtt.py

```diff
@@ -27,6 +27,8 @@ def _run_op(self, op):
             self.username = op.username
             self.client_id = op.client_id
             self.ca_cert = op.ca_cert
+            self.sas_token = None
+            self.certificate = None
```

Comment:
`x509_certificate` instead?

Reply:
resolved according to bert's comments

---
In reply to: [291739500](https://github.com/Azure/azure-iot-sdk-python-preview/pull/105#discussion_r291739500) [](ancestors = 291739500)